### DISCUSSION TITLE
How to get certificate when using Cloudflare tunnel?

### DISCUSSION THREAD

**Discussion by kyblie** [nextcloud/all-in-one#1864] [Questions] [answered]
My ISP blocks ports 80 and 443 so I have been trying to get Nextcloud running using a Cloudflare tunnel. After following the steps in the reverse-proxy documentation my browser says that there is no certificate when I try to open nextcloud itself.

I presume I might be running into the same issue as mentioned in the notes on cloudflare https://github.com/nextcloud/all-in-one/discussions/1101 However I don't see the same errors mentioning letsencrypt in my apache logs, and as I'm using the tunnel instead of just a proxy I can't disable proxying to try resolving.

**Comment by szaimen** [comment]
Hi, did you point cloudflare at the chosen apache_port? See https://github.com/nextcloud/all-in-one#how-to-run-nextcloud-behind-a-cloudflare-tunnel

**Reply by kyblie** [reply]
Yes, I have cloudflare pointed at http://localhost:11000 and the apache port also set to 11000 as suggested.

**Comment by szaimen** [comment, marked as answer]
Okay, then I am not sure what is wrong. Cloudflare should handle all the certificate stuff. So you probably need to adjust some settings in Cloudflare...

**Reply by kyblie** [reply]
Thanks, looks like something is wrong on the cloudflare side of my setup. It says the certificate has been "pending validation" for a week.
